<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>597</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>티머니 둥근바람 Regular</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>암호화폐 자동 매매 프로그램</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="mainWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>561</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="algorithm_tab">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="accessibleName">
      <string/>
     </property>
     <attribute name="title">
      <string>알고리즘</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>751</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="top_hLayout">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="blank_1">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="blank_2">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="blank_3">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="blank_4">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="add_algorithm_pushButton">
         <property name="text">
          <string>알고리즘 추가</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="delete_algorithm_pushButton">
         <property name="text">
          <string>선택 알고리즘 제거</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QListWidget" name="algorithm_listWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>751</width>
        <height>471</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="baseSize">
       <size>
        <width>5</width>
        <height>5</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>티머니 둥근바람 Regular</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="midLineWidth">
       <number>1</number>
      </property>
      <property name="autoScrollMargin">
       <number>32</number>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::NoDragDrop</enum>
      </property>
      <property name="movement">
       <enum>QListView::Static</enum>
      </property>
      <property name="isWrapping" stdset="0">
       <bool>false</bool>
      </property>
      <property name="resizeMode">
       <enum>QListView::Fixed</enum>
      </property>
      <property name="layoutMode">
       <enum>QListView::SinglePass</enum>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="gridSize">
       <size>
        <width>0</width>
        <height>60</height>
       </size>
      </property>
      <property name="viewMode">
       <enum>QListView::ListMode</enum>
      </property>
      <property name="modelColumn">
       <number>0</number>
      </property>
      <property name="uniformItemSizes">
       <bool>false</bool>
      </property>
      <property name="batchSize">
       <number>200</number>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="selectionRectVisible">
       <bool>false</bool>
      </property>
      <property name="currentRow">
       <number>-1</number>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
     </widget>
     <zorder>algorithm_listWidget</zorder>
     <zorder>horizontalLayoutWidget</zorder>
    </widget>
    <widget class="QWidget" name="project_tab">
     <attribute name="title">
      <string>프로젝트</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>751</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="top_hLayout_2">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="blank_5">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="blank_6">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="blank_7">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="log_pushButton">
         <property name="text">
          <string>거래 기록 (Log)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="add_project_pushButton">
         <property name="font">
          <font>
           <family>티머니 둥근바람 Regular</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>프로젝트 추가</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="delete_project_pushButton">
         <property name="text">
          <string>선택 프로젝트 제거</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QListWidget" name="projects_listWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>751</width>
        <height>471</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>티머니 둥근바람 Regular</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="midLineWidth">
       <number>3</number>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
